src/components/NcAppContent/NcAppContent.vue

```diff
@@ -43,6 +43,15 @@ all the children of the NcAppContent component
 	</NcAppContent>
 </template>
 ```
+```vue
+<template>
+	<NcAppContent layout="no-split">
+		<template #list>
+			<div>Full width list</div>
+		</template>
+	</NcAppContent>
+</template>
+```
```

Comment:
This example also has only one slot content, it doesn't show a switch between list and content and looks like just "Single-column".

Making a good example here could be complex to fully emulate `NcContent`. Maybe only keep the description in the prop comment and leave an example of usages for now?

Reply:
ok, im removing the example